backend/car/src/main/java/com/poongcha/car/domain/TrimRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.poongcha.car.domain;
+
+import java.util.List;
+import java.util.Optional;
+import org.springframework.data.repository.Repository;
+
+public interface TrimRepository extends Repository<Trim, Long> {
+    Trim save(final Trim trim);
+
+    Optional<Trim> findById(final long id);
+
+    List<Trim> findAllByCarType(final long id);
```

Comment:
findAllByCarTypeId()는 어떤가요?
```suggestion
    List<Trim> findAllByCarTypeId(final long id);
```

Reply:
Service에 이미 findAllByCarTypeId가 사용중이긴하네요